legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/HelperRelationalGrammarComposer.java

```diff
@@ -486,6 +492,13 @@ private static String renderViewColumnMapping(ColumnMapping columnMapping, List<
         return builder.toString();
     }
 
+    private static String renderFirstJoinPointer(JoinPointer joinPointer)
```

Comment:
lets add a comment here explaining why the first join pointer is different. 
We do this because the first join pointer requires the database pointer to be first  inside the main join operator. 
See: https://github.com/finos/legend-engine/blob/master/legend-engine-xt-relationalStore-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/RelationalParserGrammar.g4#L197